app/src/test/java/com/codesoom/assignment/application/TaskServiceTest.java

```diff
@@ -0,0 +1,203 @@
+package com.codesoom.assignment.application;
+
+import com.codesoom.assignment.TaskNotFoundException;
+import com.codesoom.assignment.models.Task;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Nested;
+import org.junit.jupiter.api.Test;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+
+@DisplayName("TaskService")
+public class TaskServiceTest {
+    private static final Long TASK_ID = 1L;
+    private static final String TASK_TITLE = "Test";
+    private static final String MODIFY_TASK_TITLE = "Modified";
+
+    private TaskService taskService;
+    private Task task;
+
+    @BeforeEach
+    void setUp() {
+        taskService = new TaskService();
+
+        task = new Task();
+        task.setTitle(TASK_TITLE);
+    }
+
+    @Nested
+    @DisplayName("getTasks")
+    class Describe_getTasks {
+        @Nested
+        @DisplayName("without any task")
+        class Context_without_any_task {
+            @Test
+            @DisplayName("return empty list")
```

Comment:
Describe Context It 패턴을 사용한다면 다음과 같이 할 수 있겠죠.
```suggestion
            @DisplayName("it returns an empty list")
```

Reply:
`@DisplayName`에 **Context**나 **Desscribe** 또 추가하는 것이 좋을까요?

Reply:
저는 굳이 추가할 이유는 없다고 생각해요.